operator/v1/types_network.go

```diff
@@ -108,6 +108,10 @@ type AdditionalNetworkDefinition struct {
 	// This must be unique.
 	Name string `json:"name"`
 
+	// namespace is the namespace of the network. This will be populated in the resulting CRD
+	// If not given the network will be created in the default namespace.
```

Comment:
Is it worth mentioning here what effect the namespace has? Can you attach pods in namespace N2 to a network in N1?

Reply:
@squeed that depends on the policy we want for OpenShift. Right now it's namespace-based, but in the future I think we'd really like to RBAC it.